babel/src/test/resources/sql/big-query.iq

```diff
@@ -2596,4 +2596,74 @@ FROM items;
 
 !ok
 
+#####################################################################
```

Comment:
can you move this section somewhere besides the end of the file? Appends cause merge conflicts